jenkins/bootstrap.py

```diff
@@ -593,6 +593,7 @@ def pr_paths(base, repos, job, build):
 GCE_KEY_ENV = 'JENKINS_GCE_SSH_PRIVATE_KEY_FILE'
 GUBERNATOR = 'https://k8s-gubernator.appspot.com/build'
 HOME_ENV = 'HOME'
+JENKINS_HOME_ENV = 'JENKINS_HOME'
```

Comment:
but prow does not set `JENKINS_HOME` right?

Reply:
Following up from https://github.com/kubernetes/test-infra/issues/4693 I definitely think we should avoid messing with `$HOME` on prow, this caused multiple bad interactions with kubetest and bazel.